docs/conf/portland/2021/news/thanks-recap.rst

```diff
@@ -0,0 +1,109 @@
+:template: {{year}}/generic.html
+:banner: _static/conf/images/headers/venue.png
+
+.. post:: May 5, 2021
+   :tags: recap, thanks, {{shortcode}}-{{year}}
+
+Conference highlights and thanks to the community!
+==================================================
+
+Hey everyone,
+
+The Write the Docs team is now emerging from the post-conference
+recovery period, and we wanted to send out a quick message to thank
+everyone for helping to make our conference such a
+success. We appreciate everyone who makes the conference possible:
+speakers, sponsors, organizers, and attendees alike.
+
+Conference recap
+----------------
+
+At **700 people** this conference was our biggest yet! We had
+the usual mix of main stage and lightning talks, unconference sessions,
+writing sprints, and social events.
+We thought everything went smoothly this year,
+including our new experiments like the new social event platform.
+
+Overall, we were excited with how our awesome community come together in our new venue.
+Lots of knowledge was shared, and we had a great time.
+
+Talk videos
+-----------
+
+Videos of the talks are now available on `a YouTube playlist`_. This
+includes all conference talks with Q&A, as well as the lightning talks.
+Thanks again to all our presenters and moderators!
+
+Huge thanks also to `Bart at Work`_ for handling the A/V production on
+our event, it was flawless once again this year. You can also subscribe
+to our `YouTube channel`_ so you don’t miss any future content.
```

Comment:
Interspersing the thanks and the subscribe is kinda awkward, I'd be tempted to ditch the call out. If folks use youtube they will know they can subscribe.

```suggestion
Videos of the talks are now available on `a YouTube playlist`_. This
includes all conference talks and Q&A, as well as the lightning talks.
You can also subscribe to our `YouTube channel`_ so you don’t miss any future content.

Thanks again to all our presenters and moderators!
Huge thanks also to `Bart at Work`_ for handling the A/V production on
our event, it was flawless once again this year.
```